### PR TITLE
Document exceptional mergeday for Firefox 57

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Just in case a similar scenario happens, I documented what needed to be done during mergeday in Firefox 57. Please let me know if something is unclear or needs to be added.

### CODE REVIEW

File: how-tos/mergeduty.md [hunk 1]
```diff
@@ -270,3 +270,36 @@ cd tools/buildfarm/maintenance/
...
+
+
+## Appendix: What to do if RelMan asks 1 release to be on 2 trains at the same time
+
+Firefox 57 was atypical: Release Management wanted it to be on both mozilla-central and mozilla-beta. The goal was to ship DevEdition 1 week before the planned merge day. This meant Releng had to:
+1. Merge m-b => m-r (regular merge per the documentation above)
+1. Merge m-c => m-b but leave m-c as is (more details below).
+1. Tag m-c to tell the beta cycle started
+1. Ask sheriffs to create a relbranch in m-b, so the last Fennec beta (56.0b13) could ship in the meantime.
+1. Once RelMan wants to bump Nighly's version number, bump and tag central.
```
Comment (srfraser) [inline how-tos/mergeduty.md]: tiny nit: typo at 'Nighly'

**Comment by srfraser** [comment]
Nicely done, great to see this documented

**Review by srfraser — APPROVED** [review]

**Review by kmoir — APPROVED** [review]
Looks good, agree with Simon's comments

**Comment by JohanLorenzo** [comment]
Thank you both!

**Pull request merged**